packages/view-types/spatial/src/ScaleBarLayer.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable */
```

Comment:
the changes of this file won't be merged in -- they are temporary to enable local testing with the changes of ScaleBarLayer. These changes will be merged as part of another PR in viv.